src/jdk.incubator.vector/share/classes/jdk/incubator/vector/X-Vector.java.template

```diff
@@ -731,6 +719,16 @@ public abstract class $abstractvectortype$ extends AbstractVector<$Boxtype$> {
                 case VECTOR_OP_URSHIFT: return (v0, v1) ->
                         v0.bOp(v1, (i, a, n) -> ($type$)((a & LSHR_SETUP_MASK) >>> n));
 #end[BITWISE]
+#if[FP]
+                case VECTOR_OP_OR: return (v0, v1) ->
```

Comment:
`VECTOR_OP_OR` looks incorrect, since `VectorOperators.OR` is not applicable to FP types.